examples/trace-model.html

```diff
@@ -0,0 +1,26 @@
+<script type="module">
+    import {HanskenClient} from './src/hansken-js.js';
+    const hansken = new HanskenClient('/gatekeeper', '/keystore');
+
+    const properties = ['uid', 'privileged', 'does.not.exist', 'picture.width', 'email.misc.flag0x0392', 'data.raw.size', 'data.raw.hash.md5', '#note.modifiedOn'];
+
+    let findProperty;
```

Comment:
can line 7 and 8 be combined. Just let findProperty = function( ...

Reply:
the function invokes itself, so it needs to be declared first to fix this